upgrade/upgradetest/config.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package upgradetest
+
+import (
+	"time"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/upgrade"
+)
+
+// GetConfig returns an upgrade config with the provided fork scheduled to have
+// been initially activated and all other forks to be unscheduled.
+func GetConfig(fork Fork) upgrade.Config {
+	return GetConfigWithUpgradeTime(fork, upgrade.InitiallyActiveTime)
+}
+
+// GetConfig returns an upgrade config with the provided fork scheduled to be
```

Comment:
preempting the sybils

```suggestion
// GetConfigWithUpgradeTime returns an upgrade config with the provided fork scheduled to be
```